frontend/src/components/grantaccess/UserIdentityCard.vue

```diff
@@ -28,29 +36,89 @@ const props = defineProps<{
-            <div class="col" style="margin-left: 2rem">
-                <label for="userId" class="row">Username</label>
-                <span id="userId" name="userId" class="row">
+            <div
+                v-if="!props.cardMgs"
+                class="col"
+                style="margin-left: 2rem"
+            >
+                <label
+                    for="userId"
+                    class="row"
+                    >Username</label
+                >
+                <span
+                    id="userId"
+                    name="userId"
+                    class="row"
+                >
                     {{ props.userIdentity.userId }}
                 </span>
             </div>
-            <div class="col" v-if="props.userIdentity.found">
-                <label for="firstName" class="row">First Name</label>
-                <span class="row" id="firstName" name="firstName">
+            <div
+                class="col"
+                v-if="props.userIdentity.found"
+            >
+                <label
+                    for="firstName"
+                    class="row"
+                    >First Name</label
+                >
+                <span
+                    class="row"
+                    id="firstName"
+                    name="firstName"
+                >
                     {{ props.userIdentity.firstName }}
                 </span>
             </div>
-            <div class="col-2" v-if="props.userIdentity.found">
-                <label for="lastName" class="row">
+            <div
+                class="col"
+                v-if="props.userIdentity.found"
+            >
+                <label
+                    for="lastName"
+                    class="row"
+                >
                     Last Name
                 </label>
-                <span id="lastName" name="lastName" class="row">
+                <span
+                    id="lastName"
+                    name="lastName"
+                    class="row"
+                >
                     {{ props.userIdentity.lastName }}
                 </span>
             </div>
-            <div class="col-6 d-flex" v-if="!props.userIdentity.found">
-                <span class="px-0 invalid" id="userNotExist">
-                    User does not exist
+            <div
+                class="col"
+                v-if="
+                    props.userIdentity.found &&
+                    props.userIdentity.businessLegalName
+                "
+            >
+                <label
+                    for="lastName"
+                    class="row"
+                >
+                    Organization Name
+                </label>
+                <span
+                    id="lastName"
+                    name="lastName"
+                    class="row"
+                >
+                    {{ props.userIdentity.businessLegalName }}
+                </span>
+            </div>
+            <div
```

Comment:
feel this logic is a little bit messed up together, user not found and error message shouldn't share the same element. And let's make sure we updated all the ids and fors (the label for organization name as well)


<img width="595" alt="Screen Shot 2024-05-03 at 3 30 12 PM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/34afa458-1dd1-452e-95c0-e151d229c1bd">